controls/V-72433.rb

```diff
@@ -77,15 +77,20 @@
 to include \"ocsp_on\"."
   tag "fix_id": "F-78785r3_fix"
 
-  describe command("grep cert_policy /etc/pam_pkcs11/pam_pkcs11.conf") do
-    its('stdout') { should include 'ocsp_on' }
-  end if smart_card_status.eql?('enabled')
 
-  describe command("grep cert_policy /etc/pam_pkcs11/pam_pkcs11.conf | wc -l") do
-    its('stdout.strip.to_i') { should cmp >= 3 }
-  end if smart_card_status.eql?('enabled')
-
-  describe "The system is not smartcard enabled" do
-    skip "The system is not using Smartcards / PIVs to fulfil the MFA requirement, this control is Not Applicable."
-  end if !smart_card_status.eql?('enabled')
+  if smart_card_status.eql?('enabled')
+    cert_policy_lines = command("grep cert_policy /etc/pam_pkcs11/pam_pkcs11.conf").stdout.split("\n")
```

Comment:
This is a nitpick, but should we use the `file` resource and pure ruby instead of `grep`?

Reply:
Yeah, just double checked and we should be able to use `inspec.file('/etc/pam_pkcs11/pam_pkcs11.conf').content.lines` instead of `grep`.

The one down side is that you have to check if content exists first I suppose.

Reply:
That's fine. I'm following that pattern in some other patches.

Reply:
I've made those changes but I've used a different pattern. The new pattern will ignore commented out lines as I was bit by this testing the updates. Check if content exists is handled by the `length >= 3` line because grep with no matches on the enumerable will return empty array.